### ISSUE TITLE
conn.close() but MS Access Laccdb (lock) file stays open

### ISSUE THREAD

**Issue by ScotterMonk** [mkleehammer/pyodbc#1279]
# Environment:
- Python 3.11 
- pyodbc latest
- OS: Windows 11
- DB: MS Access 2019
- driver: ODBC

# My issue:
Open database, read & write <-- all works fine.
At end conn.close()
.Laccdb file (MS Access lock file) stays visible in folder next to the .accdb file.
It's not a "slow" thing. I refresh, wait, etc. and that .Laccdb file is still there.
I *can* delete it manually but would like this to work the way it is supposed to.
Until I can figure out an elegant fix, I'll have my program delete the file.

**Comment by gordthompson** [comment]
Does the .laccdb file remain if you simply open the connection, wait a couple of seconds, and then close the connection?

```python
import time

import pyodbc

connection_string = "Driver={Microsoft Access Driver (*.mdb, *.accdb)};DBQ=…"
cnxn = pyodbc.connect(connection_string)
sleep_time = 3
print(f"Connected. Pausing for {sleep_time} seconds.")
time.sleep(sleep_time)
print("Closing connection.")
cnxn.close()
```

**Comment by v-chojas** [comment]
It may be a driver bug, could you post an ODBC trace? The driver is supposed to cleanup upon connection close but perhaps it isn't doing that.

**Comment by ScotterMonk** [comment]
> Also, when you say "At end conn.close()" does it mean that your script terminates immediately after the `.close()`?

Yes and I'm trying your suggestion to put a short sleep after the close.

**Comment by gordthompson** [comment]
FWIW, I am unable to reproduce your issue with ACE for Access 2013 and an .accdb on my local hard drive. The .laccdb file is removed within one second of calling `.close()` on the connection, even if the script continues to run. Is your.accdb on a network share, by chance?
